string name
float64 ctrl_steering_angle
float64 ctrl_velocity
geometry_msgs/Pose pose
sensor_msgs/Imu imu
geometry_msgs/Vector3 velocity
